src/main/java/mate/academy/repository/BookRepositoryImpl.java

```diff
@@ -0,0 +1,38 @@
+package mate.academy.repository;
+
+import java.util.List;
+import mate.academy.exception.DataProcessingException;
+import mate.academy.model.Book;
+import org.hibernate.SessionFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public class BookRepositoryImpl implements BookRepository {
+    private final SessionFactory sessionFactory;
+
+    @Autowired
```

Comment:
If you have only one constructor you can remove annotation here. 
Also, it looks consistent as you don't use this annotation in BookServiceImpl and main class

Reply:
done